src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +64,40 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+  
+  /** Returns a set of the members of this Conversation */
+  public HashSet<User> getMembers() {
+    return members;
+  }
+  
+  /** Adds given user as a member to this Conversation */
+  public void addMember(User user) {
+    members.add(user);
+  }
+  
+  @Override
+  public boolean equals(Object o) {
+    // null check
+    if (o == null) {
+      return false;
+    }
+    
+    // self check
+    if (this == o) {
+      return true;
+    }
+    
+    // class check
+    if (!(o instanceof Conversation)) {
+      return false;
+    }
+    
+    Conversation conversation = (Conversation) o;
+    return id.equals(conversation.getId());
+  }
+  
+  @Override
+  public int hashCode() {
+    return id.hashCode();
```

Comment:
Similar to above, but since I'm assuming each id is unique this should be enough. Also nice work adding equals() and hashCode()!